data/ecosystem/vendors.yaml

```diff
@@ -384,3 +384,10 @@
   contact: 'https://github.com/adnanrahic'
   oss: true
   commercial: true
+- name: Alibaba Cloud
+  distribution: true
+  nativeOTLP: true
+  url: 'https://www.alibabacloud.com/help/en/arms/tracing-analysis/get-started-with-tracing-analysis'
+  contact: ''
```

Comment:
please provide a point of contact we can reach out to in the case of questions or changes

Reply:
Thanks for reminding, I added it now.

Reply:
> please provide a point of contact we can reach out to in the case of questions or changes

Thanks for reminding, I added it now.